test/metabase/upload_test.clj

```diff
@@ -1823,16 +1824,10 @@
                           (append!)))))
                 (io/delete-file file)))))))))
 
-(defn- first-non-abstract-first-parent [value-type]
-  (cond (nil? value-type) nil
+(def ^:private column-type @#'upload/concretize)
```

Comment:
Why did you do this?

Reply:
Ah, missed this. Because the following tests call this method a lot, I made an alias rather than resolving the quote over and over again. Will rename the alias.